// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errorManagement handleError should handleError an error number 1 1`] = `
Object {
  "errorCode": 50000,
  "errorHttpCode": 500,
  "errorMessage": "Erreur serveur inconnue",
  "errorType": "Error",
  "originalException": [ServerError: error],
}
`;

exports[`errorManagement handleError should handleError an error number 2 1`] = `
Object {
  "errorCode": 50020,
  "errorHttpCode": 500,
  "errorMessage": "Erreur du serveur",
  "errorType": "Error",
  "originalException": [ServerError: error],
}
`;

exports[`errorManagement handleError should handleError an error number 3 1`] = `
Object {
  "errorCode": 50320,
  "errorHttpCode": 503,
  "errorMessage": "Le service externe a rencontré une erreur",
  "errorType": "Error",
  "originalException": [ServerError: error],
}
`;

exports[`errorManagement handleError should handleError an error number 4 1`] = `
Object {
  "errorCode": 50310,
  "errorHttpCode": 503,
  "errorMessage": "Le service externe est momentanément indisponible",
  "errorType": "Error",
  "originalException": [ServerError: error],
}
`;

exports[`errorManagement handleError should handleError an error number 5 1`] = `
Object {
  "errorCode": 50300,
  "errorHttpCode": 503,
  "errorMessage": "Le service externe a rencontré une erreur inconnue",
  "errorType": "Error",
  "originalException": [ServerError: error],
}
`;
